server/graphql/invoice/src/mutations/outbound_return/insert.rs

```diff
@@ -49,3 +76,70 @@ pub enum InsertErrorInterface {
     OtherPartyNotVisible(OtherPartyNotVisible),
     OtherPartyNotASupplier(OtherPartyNotASupplier),
 }
+
+fn map_error(error: ServiceError) -> Result<InsertErrorInterface> {
+    use StandardGraphqlError::*;
+    let formatted_error = format!("{:#?}", error);
+
+    let graphql_error = match error {
+        ServiceError::OtherPartyNotVisible => {
+            return Ok(InsertErrorInterface::OtherPartyNotVisible(
+                OtherPartyNotVisible,
+            ))
+        }
+        ServiceError::OtherPartyNotASupplier => {
+            return Ok(InsertErrorInterface::OtherPartyNotASupplier(
+                OtherPartyNotASupplier,
+            ))
+        }
+
+        // Standard Graphql Errors
+        ServiceError::InvoiceAlreadyExists => BadUserInput(formatted_error),
```

Comment:
Another small one, using `|` is a bit tidier: `ServiceError::InvoiceAlreadyExists | ServiceError::OtherPartyDoesNotExist => BadUserInput(formatted_error)`, etc..